src/sentry/static/sentry/app/components/events/interfaces/spans/spanTree.tsx

```diff
@@ -38,13 +42,69 @@ class SpanTree extends React.Component<PropType> {
     return true;
   }
 
+  generateInfoMessage(input: {
+    isCurrentSpanHidden: boolean;
+    numOfSpansOutOfViewAbove: number;
+    isCurrentSpanFilteredOut: boolean;
+    numOfFilteredSpansAbove: number;
+  }): React.ReactNode {
+    const {
+      isCurrentSpanHidden,
+      numOfSpansOutOfViewAbove,
+      isCurrentSpanFilteredOut,
+      numOfFilteredSpansAbove,
+    } = input;
+
+    const messages: React.ReactNode[] = [];
+
+    const showHiddenSpansMessage = !isCurrentSpanHidden && numOfSpansOutOfViewAbove > 0;
+
+    if (showHiddenSpansMessage) {
+      messages.push(
+        <span key="spans-out-of-view">
+          <strong>{t('Spans out of view:')}</strong> {numOfSpansOutOfViewAbove}
```

Comment:
Should this be more natural language like `%d spans out of view`?

Reply:
done.